editoast/src/views/temporary_speed_limits.rs

```diff
@@ -111,18 +111,32 @@ enum TemporarySpeedLimitError {
     #[error("Name '{name}' already used")]
     #[editoast_error(status = 400)]
     NameAlreadyUsed { name: String },
+    #[error(transparent)]
+    #[editoast_error(status = 500)]
+    Database(model::Error),
 }
 
-fn map_diesel_error(e: InternalError, name: impl AsRef<str>) -> InternalError {
-    if e.message.contains(
-        r#"duplicate key value violates unique constraint "temporary_speed_limit_group_name_key""#,
-    ) {
-        TemporarySpeedLimitError::NameAlreadyUsed {
-            name: name.as_ref().to_string(),
+impl From<model::Error> for TemporarySpeedLimitError {
```

Comment:
Will we have to write all `From` manually from now on? Or is it just a temporary situation until we finish the migration to the new error model (I feel I'm missing a part of the big picture here). If it's the former, then we might to discuss it because I'm not sure we would want that everywhere, what do you think?